src/consumeUntil.js

```diff
@@ -211,7 +211,7 @@ export default async function consumeUntil (target_content_height, container, op
 				...options,
 				startAtIndex: i + 1,
 				// We cannot shift beyond a heading with level <= of heading or another shiftable
-				stopAt: n => util.getHeadingLevel(n.nodeName) >= headingLevel || util.isShiftable(n, options),
+				stopAt: n => util.getHeadingLevel(n) >= headingLevel || util.isShiftable(n, options),
```

Comment:
I just committed this directly so we have time to iterate on the rest

Reply:
I'll commit it directly and update the PR from `main`.

Reply:
Thanks!